config/environments/production.rb

```diff
@@ -70,7 +70,7 @@
     config.action_mailer.perform_deliveries = false
   end
 
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
```

Comment:
When `perform_deliveries` is set to `false` (when SMTP isn't used),
Does the mailer will keep throwing exceptions then?
If yes, then I think this will block us from using GL3 without SMTP configured.

Reply:
Nope - it just skips over it:
```
Skipped delivery of mail 63ea81b08a2de_d502ba7c307e4@ahmad-computer.mail as `perform_deliveries` is false
```